connexion/app.py

```diff
@@ -87,7 +87,8 @@ def common_error_handler(exception):
         return problem(title=exception.name, detail=exception.description, status=exception.code)
 
     def add_api(self, swagger_file, base_path=None, arguments=None, auth_all_paths=None, swagger_ui=None,
-                swagger_path=None, swagger_url=None, validate_responses=False, resolver=Resolver()):
+                swagger_path=None, swagger_url=None, validate_responses=False, resolver=Resolver(),
+                security_decorator=None):
```

Comment:
the app constructor need not know about the decorator here. The decorator will be specific to an API that is added using the `app.add_api()` AFTER the app was instantiated using 

`app = connection.App(....)`